Update release-notes.md (#797)
GovTechSG/seed-documentation: release-notes.md

---

## 29 August 2024

| <div style="width:100px"><b>Type</b></div> | <div style="width:150px"><b>Change</b></div> | **Description** |
|---|---|---|
| **Update** | **Cloudflare WARP client update** | Updated Cloudflare WARP client versions: <br>- Windows: 2024.6.473.0 <br>- macOS: 2024.6.474.0 <br><br>**Impact:** Users may experience a brief disconnection from SGTS resources during the installation process, lasting up to five minutes. No action is required if your device already has the updated version. <br><br>For assistance, contact **enquiries_seed@tech.gov.sg**. |

---

## 22 August 2024

| <div style="width:100px"><b>Type</b></div> | <div style="width:150px"><b>Change</b></div> | **Description** |
|---|---|---|
| **Bug fix** | **macOS 12 onboarding blocked** | Devices running macOS 12 are now blocked from onboarding to SEED. <br><br>For assistance, contact **enquiries_seed@tech.gov.sg**. |

---

## 14 August 2024

| <div style="width:100px"><b>Type</b></div> | <div style="width:150px"><b>Change</b></div> | **Description** |
|---|---|---|

> | **Update** | **Email notifications and logging enhancements** | Standardised device naming in email notifications and added Tanium computer IDs to CloudWatch logs for `query-tanium-eid-last-seen` lambda. <br><br>For assistance, contact **enquiries_seed@tech.gov.sg**. |

Reviewer: ```suggestion
| **Bug fix** | **Email and desktop notifications** | Device naming will be standardised for notifications <br><br>For assistance, contact **enquiries_seed@tech.gov.sg**. |
```

Reviewer: Just changing this. The internal portion is not relevant to users.

---

## 7 August 2024

| <div style="width:100px"><b>Type</b></div> | <div style="width:150px"><b>Change</b></div> | **Description** |
|---|---|---|

> | **Update** | **SEED dashboard UI changes** | Updated SEED dashboard with new configurations for DBB checks. Explore the changes on [SEED Dashboard tour](https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/seed-dashboard/seed-dashboard-tour) <br><br>For assistance, contact **enquiries_seed@tech.gov.sg**. |

Reviewer: ```suggestion
| **Update** | **SEED dashboard UI changes** | The SEED Dashboard has been updated with information on SEED Configuration issues. Explore the changes on [SEED Dashboard tour](https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/seed-dashboard/seed-dashboard-tour) <br><br>For assistance, contact **enquiries_seed@tech.gov.sg**. |
```
